CONTRIBUTING.md

```diff
@@ -139,12 +136,13 @@ using your real name (sorry, no pseudonyms or anonymous contributions.)
 
 You can add the sign off when creating the git commit via `git commit -s`.
 
-[code-of-conduct]: https://github.com/opencontainers/tob/blob/d2f9d68c1332870e40693fe077d311e0742bc73d/code-of-conduct.md
+[code-of-conduct]: https://github.com/opencontainers/org/blob/master/.github/CODE_OF_CONDUCT.md
 [developercertificate.org]: http://developercertificate.org/
 [Freenode]: https://freenode.net/
 [irc-logs]: http://ircbot.wl.linuxfoundation.org/eavesdrop/%23opencontainers/
 [mailing-list]: https://groups.google.com/a/opencontainers.org/forum/#!forum/dev
 [meeting.ics]: https://github.com/opencontainers/runtime-spec/blob/master/meeting.ics
 [minutes]: http://ircbot.wl.linuxfoundation.org/meetings/opencontainers/
 [rfc5545]: https://tools.ietf.org/html/rfc5545
+[security]: https://github.com/opencontainers/org/blob/master/security
```

Comment:
/s/security/security.md#reporting-security-issues/

Reply:
We just decided on the call to rename the code-of-conduct / change its location, so I think we should wait for https://github.com/opencontainers/org/pull/6 to be approved / merged and then I can update all links here.